starter-code/books-coffee/views/show.ejs

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>coffee</title>
+    <link rel='stylesheet' href='/stylesheets/style.css' />
+  </head>
+  <body>
+
+<%= place.name %>
+<hr>
+<%= place.type %>
+<hr>
+Latitude: <%= place.location.coordinates[1] %> | Longitude: <%= place.location.coordinates[0] %>
+
+<div id="map"></div>
+
+
+<script
+src="https://code.jquery.com/jquery-3.2.1.min.js"
+integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
+crossorigin="anonymous"></script>
+<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDUeQXCyJDlhOtCB8JwWAk8zCxpjk6k-jo"></script>
+<script src="/javascripts/main.js" type="text/javascript"></script>
+<script type="text/javascript">
+  var myPlaces = <%- JSON.stringify(place) %>
```

Comment:
I get a `myPlaces.forEach` is not a function because in the `main.js` you expect the map to display multiple markers. However, as this page is for a single place, your are passing only one.

The smallest effort that will get your code working is `myPlaces = [<%- JSON.stringify(place) %>]`.
Ideally you would have different Javascript functions for different files.